File: src/test/__samples__/search/cpp/foo.cpp
#include <stdio.h> 
void foo(int x) 
{ 
   x = 30; 
} 
  
int main(void) 
{ 
    int x = 20; 
    foo(x); 
    printf("x = %d", x); 
    return 0; 
}  

File: doc/search/py/foo.py
for i in range(1, 101):
    if i % 15 == 0:
        foo = "FizzBuzz"
    elif i % 3 == 0:
        foo = "Fizz"
    elif i % 5 == 0:
        foo = "Buzz"
    else:
        foo = i